pkgs/development/tools/rust/cargo-pgx/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, pkg-config, openssl, libiconv, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-pgx";
+  version = "0.4.5";
+
+  src = fetchFromGitHub {
+    owner = "tcdi";
+    repo = "pgx";
+    rev = "v${version}";
+    sha256 = "sha256-M81n4ORhMYpXRonJxEgFMOuR0Deb5CN2df70ltviIqI=";
+  };
+
+  cargoSha256 = "sha256-8x2oTnFWDx+57OKamci78WRPh+dVBj6SKJALlOBHuSg=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ]
+    ++ lib.optionals stdenv.isDarwin [ libiconv Security ];
```

Comment:
The `libiconv` dependency can be dropped here -- it's [implicit on Darwin](https://github.com/NixOS/nixpkgs/blob/cff945d97b31d343f0d620c7fb0806d85b713c4a/pkgs/build-support/rust/build-rust-package/default.nix#L130).

Reply:
done